go/props/page.go

```diff
@@ -2,6 +2,21 @@ package props
 
 import "github.com/a-h/templ"
 
+// PageHeader Templ component props
+type PageHeader struct {
+	Title       string
+	Class       string
+	Attrs       templ.Attributes
+	Breadcrumbs []PageBreadcrumb
+}
+
+// PageBreadcrumb defines a Page Breadcrumb to be displayed on a PageHeader
```

Comment:
This comment would be a good place to add details about when the Attrs would be used.